pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/LmsInsertEvents.java

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.ddp.studybuilder.task;
+
+import lombok.extern.slf4j.Slf4j;
+import one.util.streamex.StreamEx;
+import org.broadinstitute.ddp.db.dao.EventDao;
+import org.broadinstitute.ddp.db.dao.JdbiEventConfiguration;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.model.activity.types.EventActionType;
+import org.broadinstitute.ddp.model.activity.types.EventTriggerType;
+import org.broadinstitute.ddp.model.dsm.DsmNotificationEventType;
+import org.broadinstitute.ddp.model.event.DsmNotificationTrigger;
+import org.broadinstitute.ddp.model.event.EventConfiguration;
+import org.jdbi.v3.core.Handle;
+
+@Slf4j
+public class LmsInsertEvents extends InsertStudyEvents {
+    public LmsInsertEvents() {
+        super("cmi-lms", "patches/lms-new-events.conf");
+    }
+
+    @Override
+    public void run(final Handle handle) {
+        removeExistingEvents(handle);
+        super.run(handle);
+    }
+
+    private void removeExistingEvents(final Handle handle) {
```

Comment:
I believe adding a cancel expr to existing event would have been little more cleaner rather than disabling existing event and re-inserting new event but this approach works too.
Just leaves a disabled event config row in there.